samples/ConsoleApp.SelfBootstrap/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading.Tasks;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace ConsoleApp.SelfBootstrap
+{
+    internal class Program
+    {
+        private static readonly ActivitySource MyActivitySource = new ActivitySource("OpenTelemetry.AutoInstrumentation.ConsoleApp");
```

Comment:
Here it will be better to have something like the name of the app or assembly. Making it automatically match with the auto-instrumentation won't be the typical scenario.

Reply:
Done